player/public/index.html

```diff
@@ -74,6 +74,8 @@
     <script type="text/javascript" src="js/splashScreen.js"></script>
     <script type="text/javascript" src="js/main.js"></script>
     <script type="text/javascript" src="js/app.js"></script>
+    <script type="text/javascript" src="js/webView.js"></script>
+    <script type="text/javascript" src="js/mobileView.js"></script>
```

Comment:
There should be localView.js file should load only when we are launching locally.